cf-agent/verify_files_utils.c

```diff
@@ -1495,7 +1509,7 @@ bool CopyRegularFile(EvalContext *ctx, const char *source, const char *dest, str
     if (rsrcfork)
     {                           /* Can't just "mv" the resource fork, unfortunately */
         rsrcrd = safe_open(new, O_RDONLY | O_BINARY);
-        rsrcwd = safe_open(dest, O_WRONLY | O_BINARY | O_CREAT | O_TRUNC, 0600);
+        rsrcwd = safe_open(dest, O_WRONLY | O_BINARY | O_CREAT | O_TRUNC);
```

Comment:
@karlhto It might be easier/safer to just backport, just adapt your changes with what is in 3.12.x. (i.e. don't try to backport `safe_[f]open` function changes).